### PR TITLE
Added some resources to free-courses-id.md

### PR DESCRIPTION
**Pull request by adipurnamk**
added some resources in Bahasa Courses
 - Docker
 - git

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-id.md [hunk 1]
```diff
@@ -111,6 +112,8 @@
 ### Docker
 
 * [Belajar Docker untuk Pemula](https://www.youtube.com/playlist?list=PL4SGTPmSY0qkxCTe3Gd0wA-bQZChXhsNI) - Giri Kuncoro (YouTube)
+* [Mengenal Container dan Docker Sampai Jago Dalam 2 Jam](https://www.youtube.com/watch?v=26O6Ke03j3Y) - Imre Nagi, Cloud Engineering with Imre (YouTube)
```
Comment (shrn01) [inline courses/free-courses-id.md:115]:
```suggestion
* [Mengenal Container dan Docker Sampai Jago Dalam 2 Jam](https://www.youtube.com/watch?v=26O6Ke03j3Y) - Imre Nagi
```

File: courses/free-courses-id.md [hunk 2]
```diff
@@ -111,6 +112,8 @@
...
+* [Pengembangan Microservice Dengan Docker Compose](https://www.youtube.com/watch?v=ALGVV5cGUtc) - Imre Nagi, Cloud Engineering with Imre (YouTube)
```
Comment (shrn01) [inline courses/free-courses-id.md:116]:
```suggestion
* [Pengembangan Microservice Dengan Docker Compose](https://www.youtube.com/watch?v=ALGVV5cGUtc) - Imre Nagi
```

File: courses/free-courses-id.md [hunk 3]
```diff
@@ -396,6 +400,11 @@
 * [Tutorial Blockchain Bahasa Indonesia](https://www.youtube.com/playlist?list=PLH1gH0TmFBBhvZi4kEqU6kCjyv_y8qBae) - Ipung DEV Academy (YouTube)
 
 
+### Terraform
+
+* [Bajalar Terraform untuk Pemula](https://www.youtube.com/watch?v=Ce4Q1xLDNjE&list=PL4SGTPmSY0qngs44Ssc0RHO9h4fmZ9JUb) - Giri Kuncoro (YouTube)
```
Comment (shrn01) [inline courses/free-courses-id.md:405]: Please add playlist link instead of the video link

File: courses/free-courses-id.md [hunk 4]
```diff
@@ -139,6 +142,7 @@
 * [Belajar GIT (Source Code Management)](https://www.youtube.com/playlist?list=PL8bBYpHH3RI6BlCzFTMQvt7sGSycUj7S-) - Eka Putra, UpKoding (YouTube)
 * [GIT & GITHUB](https://www.youtube.com/playlist?list=PLFIM0718LjIVknj6sgsSceMqlq242-jNf) - Sandhika Galih, Web Programming UNPAS (YouTube)
 * [Git Tutorial - Bahasa Indonesia](https://www.youtube.com/playlist?list=PL-CtdCApEFH_lYGV8hxqjtKmFA_xeLupq) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
+* [Jago Git Dalam 2 Jam](https://www.youtube.com/watch?v=KGSfUgaiVNI) - Imre Nagi, Cloud Engineering with Imre (YouTube)
```
Comment (shrn01) [inline courses/free-courses-id.md:145]:
```suggestion
* [Jago Git Dalam 2 Jam](https://www.youtube.com/watch?v=KGSfUgaiVNI) - Imre Nagi
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-id.md (hunk 3 above)
Comment (adipurnamk) [reply courses/free-courses-id.md]: Hello @shrn01, thanks for pointing these out. 
I have fixed it in my latest PR.

File: courses/free-courses-id.md [hunk 5]
```diff
@@ -397,6 +401,11 @@
 * [Tutorial Blockchain Bahasa Indonesia](https://www.youtube.com/playlist?list=PLH1gH0TmFBBhvZi4kEqU6kCjyv_y8qBae) - Ipung DEV Academy (YouTube)
 
 
+### Terraform
+
+* [Belajar Terraform untuk Pemula](https://www.youtube.com/playlist?list=PL4SGTPmSY0qngs44Ssc0RHO9h4fmZ9JUb) - Giri Kuncoro (YouTube)
```
Comment (shrn01) [inline courses/free-courses-id.md:406]:
```suggestion
* [Belajar Terraform untuk Pemula](https://www.youtube.com/playlist?list=PL4SGTPmSY0qngs44Ssc0RHO9h4fmZ9JUb) - Giri Kuncoro
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**